cmd/karavictl/cmd/testdata/kubectl_get_secret_storage_powerflex_powermax.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Is the plan to have all supported storage system test data in this one file? We might want to consider removing the storage system names from the file name if that's the case

Reply:
There is an existing file called `kubectl_get_secret_storage.json` with just one PowerFlex system. I created this separate file specifically for having a PowerFlex and PowerMax system incase we want to test with various secrets.